### DISCUSSION TITLE
Why should seaweedfs offset be divisible by 8?

### DISCUSSION THREAD

**Discussion by mynamebvh** [seaweedfs/seaweedfs#4209] [Q&A] [answered]
I read the source code of seaweedfs, I wonder why offset must be divisible by 8 ?

**Comment by chrislusf** [comment, marked as answer]
this will amplify the 4-byte offset addressing space by 8 times. 

you may read the facebook haystack paper.

**Reply by mynamebvh** [reply]
Thanks
